### PR TITLE
Removing traces of rollback update

### LINKED ISSUE #846: Remove deprecated Rollback
**Is your feature request related to a problem? Please describe.**
Stale rollback code still present in the codebase

**Describe the solution you'd like**
Remove the old rollback from the ADMIN proto, admin server and move back the test to standard rollback API.

### PR DESCRIPTION
**Pull request by Andrea-Campanella**
Fixes #846

### CODE REVIEW

File: pkg/cli/rollback.go [hunk 1]
```diff
@@ -20,17 +20,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func getRollbackNewCommand() *cobra.Command {
+func getRollbackCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "rollback-new <changeId>",
 		Short: "Rolls-back a new network change",
```
Comment (SeanCondon) [inline pkg/cli/rollback.go]: remove `new` from the `Use` and `Short`.

File: pkg/northbound/admin/admin.go [hunk 2]
```diff
@@ -207,10 +207,15 @@ func (s Server) ListRegisteredModels(req *ListModelsRequest, stream ConfigAdminS
 	return nil
 }
 
-// RollbackNewNetworkChange rolls back a named new (atomix-based)network changes.
+<<<<<<< Updated upstream
```
Comment (SeanCondon) [inline pkg/northbound/admin/admin.go]: this does not look good - merge error?

File: pkg/cli/rollback.go (hunk 1 above)
Comment (Andrea-Campanella) [reply pkg/cli/rollback.go]: done

File: pkg/northbound/admin/admin.go (hunk 2 above)
Comment (Andrea-Campanella) [reply pkg/northbound/admin/admin.go]: IDK what happened there. fixing

**Pull request merged**